app/controllers/courses_controller.rb

```diff
@@ -15,16 +13,15 @@ def create
       redirect_to course_show_path(@course.id)
       return
     else
-      render 'new'
-      return
+      render 'new' && return
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.